src/packer.pkr.hcl

```
@@ -59,15 +59,15 @@ data "amazon-ami" "debian_bullseye" {
 
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
-source "amazon-ebs" "example" {
+source "amazon-ebs" "egress-assess" {
```

We prefer snake case for HCL:
```suggestion
source "amazon-ebs" "egress_assess" {
```